content/S2016/S-16-event-square.md

```diff
@@ -0,0 +1,20 @@
+title: Square Tech Workshop: Becoming Ok with Okhttp
+Date: 2016-05-18 
+Category: Events 
+Tags: academic 
+Slug: square-event-s16 
+Author: Bhargavi Dameracharla 
+Summary: Technical workshop on open source HTTP & HTTP/2 client
+
+Square is holding a technical workshop on their open source HTTP & HTTP/2 client for Android and Java applications. 
+Join us to learn more about Okhttp and get a chance to speak with their engineers! Food will be available. 
+(Including vegetarian, vegan, and gluten-free options)
+There is an attendance cap of 70 people (50 undergrads and 20 grads), so make sure to [register]( https://www.eventbrite.com/e/square-tech-workshop-becoming-ok-with-okhttp-tickets-25360009471) early!
```

Comment:
There is an extra space before the link on this line. Can you remove that? and maybe put it on its own line since this line is quite long and wraps around.

You'll also need to add another newline for these two lines to constitute separate paragraphs; you can see on the deployed review app that this post is currently one long paragraph.